### ISSUE TITLE
Evaluate displaying of playlist items as albums

### ISSUE THREAD

**Issue by digimezzo** [digimezzo/dopamine#215]
This is a user's request:

Dopamine lacks is the option to open a playlist and view the contents represented as album covers. I often make playlists of ALBUMS that I group together but I can only view the playlist as a long list of tracks.

Just as an example of what I mean, I made this playlist on iTunes with a few random songs, then selected "view as albums" in the options bar.

![image](https://user-images.githubusercontent.com/11945339/153156350-0c7f3906-7382-4b08-8978-d34662cb6e0a.png)

**Comment by Not-Sure-Whats-Going-On** [comment]
Yes please!  This would make a great way of navigating a sunset of large collection.
